libs/cardano-ledger-core/src/Cardano/Ledger/BHeaderView.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+
+module Cardano.Ledger.BHeaderView where
+
+import Cardano.Ledger.BaseTypes (BoundedRational (..), UnitInterval)
+import Cardano.Ledger.Hashes (EraIndependentBlockBody)
+import Cardano.Ledger.Keys (Hash, KeyHash, KeyRole (..))
+import Cardano.Ledger.Slot (SlotNo (..), (-*))
+import Numeric.Natural (Natural)
+
+data BHeaderView crypto = BHeaderView
```

Comment:
I think it would be helpful to add some explanatory documentation here!

Reply:
yes indeed, thanks!